wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -687,6 +849,19 @@ instance Arbitrary PaymentDistribution where
   arbitrary = PaymentDistribution <$> arbitrary
                                   <*> arbitrary
 
+deriveSafeBuildable ''PaymentDistribution
+instance BuildableSafeGen PaymentDistribution where
+    buildSafeGen sl PaymentDistribution{..} = bprint ("{"
+        %" address="%buildSafe sl
+        %" amount="%buildSafe sl
+        %" }")
+        pdAddress
+        pdAmount
+
+-- instance Buildable [PaymentDistribution] where
+--     build = bprint listJson
```

Comment:
Should we delete this commented-out code?

Reply:
Whoops. Will do.